src/pages/workspaces/workspace/Data.js

```diff
@@ -390,6 +387,14 @@ const DataTableActions = ({ workspace, tableName, rowCount, entityMetadata, onRe
   ])
 }
 
+const workspaceDataTypes = {
+  entities: 'entities',
+  snapshot: 'snapshot',
+  referenceData: 'referenceData',
+  localVariables: 'localVariables',
+  bucketObjects: 'bucketObjects'
+}
```

Comment:
It took me a bit to recognize that this is really just helping to avoid misspelling the data types. Given that level of importance, I might compress the space used a bit:
```suggestion
const workspaceDataTypes =
  'entities snapshot referenceData localVariables bucketObjects'.split(/\s+/).reduce((acc, k) => ({...acc, [k]: k}), {})
```
This also has the advantage of ensuring the keys and values are indeed identical at the expense of taking more time to understand. I don't have a strong preference, just highlighting the tradeoffs for you to make the decision.

Reply:
We have a couple of these enum like objects in various places. Perhaps it's worth adding a utils function to generate them?

Reply:
Added a utils function in [ec3187e](https://github.com/DataBiosphere/terra-ui/pull/3316/commits/ec3187ec4c7f6b2b2ae8073dccfe8757f2618919).